hudi-common/src/main/java/org/apache/hudi/common/table/log/AbstractHoodieLogRecordScanner.java

```diff
@@ -244,7 +244,7 @@ public void scan() {
       progress = 1.0f;
     } catch (Exception e) {
       LOG.error("Got exception when reading log file", e);
-      throw new HoodieIOException("IOException when reading log file ");
+      throw new HoodieIOException("IOException when reading log file ", e);
```

Comment:
@lw309637554 Can you check if the catch (Exception e) can be changed to catch (IOException io) ? If not that we are not just catching IOException but also other exceptions and then we should add appropriate multiple catch blocks and throw the correct exception.

Reply:
ok , i will see it.

Reply:
can not change to IOException, because in processQueuedBlocksForInstant , the implement for LogRecordScannerCallback will throw InterruptedException exception, may be other implement throw other exception.
  public static interface LogRecordScannerCallback {

    public void apply(HoodieRecord<? extends HoodieRecordPayload> record) throws Exception;
  }.

so , i think can catch also IOException wrap HoodieIOException and Exception wrap HoodieException